ompi/runtime/ompi_mpi_params.c

```diff
@@ -263,12 +263,11 @@ int ompi_mpi_register_params(void)
         ompi_rte_abort(1, NULL);
     }
 
-    ompi_add_procs_cutoff = 1024;
```

Comment:
@hjelmn and I chatted on the phone -- he filled me in on what I was missing: there's a possible inconsistency in behavior with a program that does something like this:

```C
MPI_T_Init(...);
MPI_T_Cvar_read(..., &value_pre);
MPI_T_Cvar_write(...);
MPI_T_Finalize(...);

MPI_Init(...)
MPI_T_Init(...);
MPI_T_Cvar_read(..., &value_post);
assert(value_pre == value_post);
```

When the cvar is part of a dynamically-loaded component, the `assert` will be true. If we remove the additional assignment, when the cvar is part of a statically-loaded component, the `assert` will be false.  Nathan's second assignment ensures that the `assert` will be true in both cases.

This is actually a larger question for the MPI Tools Working Group in the MPI Forum: what consistency guarantees -- if any -- are provided by the MPI_T API when MPI_T and/or MPI is finalized?

For this PR, I'll remove my deletion of the 2nd assignment, and we'll leave OMPI's behavior in this area as it was.  Nathan and I will bring up this what-does-MPI_T-guarantee issue with the Tools WG separately.